app/pages/home-for-user/index.jsx

```diff
@@ -281,8 +281,12 @@ export default class HomePageForUser extends React.Component {
                 onClose={this.deselectSection}
               />
             )}
-          </div>
 
+            <div className="home-page-for-user__down-arrow">
+              <i className="fa fa-arrow-down fa-2x" aria-hidden="true" />
+            </div>
+
+          </div>
```

Comment:
It seems a bit unnecessary here, code-wise, to have a `div` with one item in it when you could have the icon standalone. However, this isn't blocking.